tests/framework/installer/ceph_manifests.go

```diff
@@ -1603,7 +1610,11 @@ spec:
     config:
       storeType: "` + settings.StoreType + `"
       databaseSizeMB: "1024"
-      journalSizeMB: "1024"`
+      journalSizeMB: "1024"
+  mgr:
+    modules:
+    - name: pg_autoscaler
```

Comment:
Are there any tests that use mimic that this will cause to fail?

Reply:
I'll double check on that.

Reply:
The errors for any of the modules actually don't fail the orchestration (see [here](https://github.com/rook/rook/blob/master/pkg/operator/ceph/cluster/mgr/mgr.go#L181-L191)). They only show up as errors in the log. This falls in the category of errors that we should be surfacing in the cluster status, which we need to do. I'll open an issue for that.